### DISCUSSION TITLE
error when using accession number

### DISCUSSION THREAD

**Discussion by kcamnairb** [antismash/antismash#590] [General]
Hi,  when I use an accession number as the data input for the antismash fungal version i get an error "Failed to download file from NCBI". I'm using the accession number GCA_009017415.1 . Is this the correct number to be using in this box?

Thanks,
Brian

**Comment by kblin** [comment]
We're using the NCBI Entrez API to download sequences, and for some reason that unfortunately doesn't support assembly IDs.

**Reply by kcamnairb** [reply]
How would one find the accession number for a genome assembly? This is a bit confusing because the "assembly ID", GCA_009017415.1 is actually labelled as the "GenBank assembly accession" on [NCBI](https://www.ncbi.nlm.nih.gov/assembly/GCA_009017415.1).
